.github/workflows/version.yml

```diff
@@ -0,0 +1,25 @@
+name: Version
```

Comment:
can we run this CI workflow only when either Makefile or package.json is touched by the commit/PR?

We should not have to run this for every commit/PR since it will be rare Makefile or package.json will be changing.

Reply:
I think that yes we can